docs/config.rst

```diff
@@ -710,6 +729,21 @@ Features
 
    .. versionadded:: 2.2
 
+.. attribute:: LP_ENABLE_MUX
+   :type: bool
+   :value: 1
+
+   Allows getting the name of the current multiplexer session
+   (*screen* or *tmux*).
+
+   If set to ``0``, also disables:
+
+   * :attr:`LP_COLOR_IN_MULTIPLEXER`,
+   * :attr:`LP_MARK_MULTIPLEXER_OPEN` and :attr:`LP_MARK_MULTIPLEXER_CLOSE`,
+   * :attr:`LP_ENABLE_SCREEN_TITLE` and :attr:`LP_ENABLE_TMUX_TITLE_PANES`.
```

Comment:
Hm, this is a difficult one. As your docs above describe, disabling this will prevent display of the type of multiplexer and any marks that would show up.

But setting the title correctly depends on the detection of multiplexers. I would not want that to break if I did not want to see multiplexer related info.

I am not sure how best to fix this. Maybe a new flag like `--force` to `_lp_multiplexer()` that will make it ignore the config option that we can use on [line 844](https://github.com/liquidprompt/liquidprompt/blob/1f7858af421e0c8c45134b509b774842c08d17de/liquidprompt#L844C16-L844C16)?

Reply:
I am not a big fan of adding options, but what about splitting the feature in two: one for the multiplexer display, and one for the multiplexer title, and leave the detection on?

Reply:
Don't we have that already with the `LP_ENABLE_SCREEN_TITLE` option? That is what I mean; a user would expect the `LP_ENABLE_SCREEN_TITLE` to just work, and not be tied to `LP_ENABLE_MUX`.

Reply:
Indeed, that would mean to just add something like `LP_ENABLE_MULTIPLEXER_MARKS` (to avoid the unclear link with titles).

Reply:
So you are proposing we would have three config options?
* `LP_ENABLE_MUX`
* `LP_ENABLE_SCREEN_TITLE` (only enabled if `LP_ENABLE_MUX` is)
* `LP_ENABLE_MULTIPLEXER_MARKS` (only enabled if `LP_ENABLE_MUX` is)

Why do we need `LP_ENABLE_MUX` in that case? I say make `LP_ENABLE_MUX` instead function like `LP_ENABLE_MULTIPLEXER_MARKS` and we make it completely separate from `LP_ENABLE_SCREEN_TITLE`.

Reply:
Yes, we agree. I gave it a try in 6eb8eb6.